lib/matplotlib/__init__.py

```diff
@@ -1115,7 +1115,7 @@ def rc_params_from_file(fname, fail_on_error=False, use_default_template=True):
     if config['datapath'] is None:
         config['datapath'] = get_data_path()
 
-    if not config['text.latex.preamble'] == ['']:
+    if "".join(config['text.latex.preamble']):
```

Comment:
Not sure it is any clearer.
Ultimately I think config["text.latex.preamble"] should just be a multiline string (then we'd just check that it is empty after calling strip()) but that'd require hacking such a type into rcsetup.py which I don't really want to do (as discussed many times I believe the format of rcsetup to be silly).